kpet/utils.py

```diff
@@ -33,3 +37,106 @@ def get_template_content(filename, dbdir=None):
     template_path = get_template_path(filename, dbdir)
     with open(template_path) as file_handler:
         return file_handler.read()
+
+
+def print2(s):
+    sys.stderr.write(s + '\n')
+
+
+def smart_layout_insert_prefix(f_layout, obj):
+    if obj is None:
+        return None
+    prefix = os.path.dirname(os.path.realpath(f_layout))
+    return '%s/%s' % (prefix, str(obj))
```

Comment:
It seems that you are doing the same job that `os.path.join` does. At least there is a strong reason for not using it, please use the standard modules as much as possible.

Reply:
Well, use `os.path.join` instead.